sketchcrapp.sh

```diff
@@ -312,16 +313,18 @@ https://github.com/duraki/SketchCrapp"
       testBundleVersionString="68.1"
       ;;
     *)
-      echo "[+] Copy the details below and open a new issue on GitHub repository: \
+      testBundleVersionString="binaryerr››"
+      echo "[ERR] Hash invaild, can’t look up version from hash."
+      echo "[INFO] Carefully review README file again"
+      echo "[INFO] If you still have problem copy the details below and open a new issue on GitHub repository: \
 https://github.com/duraki/SketchCrapp"
       echo "+==================================================================="
-      echo "+ Issue details ‹s:sketchcrapp›"
       echo "+ Application Path  : $appPath"
       echo "+ Application Binary: $execPath"
       echo "+ Passed version    : $bundleVersionString"
       echo "+ Correct hash      : $(getHashFromVersionString "$bundleVersionString")"
       echo "+ Binary SHA1       : $appSHA1"
-      echo "+ Error             : Can’t look up version from hash."
+      echo "+ Error             : Hash invaild" 
```

Comment:
I would leave it as `Can’t look up version from hash.`. It's more informative then just "Hash Invalid" (doesn't make sense).

Reply:
Or better yet; `Can't find Sketch with that signature. Hash is invalid.`